gnomic/genotype.py

```diff
@@ -261,7 +261,7 @@ def changes(self):
 class Genotype(object):
     def __init__(self, changes, parent=None):
         if parent:
-            state = GenotypeState(parent.state.changes)
```

Comment:
It actually expects Change objects. The bug here was that it's not transforming them:

```
class GenotypeState(object):
    _changes: List[Tuple[Union[Annotation, AtLocus, None], Union[Annotation, None]]]
    def __init__(self, changes: Sequence[Change] = ()) -> None:
        ...
```

`GenotypeChange._changes` is private.